home.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
```

Comment:
By default, we should name the home page as index.html - index for any file structure will always be seen to be the root of the project file.